### ISSUE TITLE
Cases are inconsistent for table and column names in the SQL code

### ISSUE THREAD

**Issue by traber** [bitcoinj/bitcoinj#1055]
The SQL statements declared in the class DatabaseFullPrunedBlockStore and MySQLFullPrunedBlockStore are inconsistent. Sometimes the names are kept in camel case (e.g. "openOutputs" or "scriptBytes") and sometimes they are not (e.g. "openoutputs" or "scriptbytes"). This might work on a Windows system but not on a UNIX system. In consequence the current implementation is resulting in MySQLSyntaxErrorExceptions being thrown while creating or using a MySQLFullPrunedBlockStore with a database running on a UNIX system.

**Comment by kparmar1** [comment]
Hmmm yes you are correct...inconsistencies.  I think it would be better 
to change all to lower case?
Any objections?

This would keep it all consistent with Postgres store (I just checked 
and it uses lowercase) (I haven't ever used cases for column names 
before, usually a _ for rational DB's anyway).

I didn't realise there were inconsistent names between Postgres and H2 
(case wise) when I factored out the code (I must have used H2 as the 
guild for MySQL and not Postgres )- My Bad - I should have noticed that 
(OSx obviously more like windows although I would have expected the 
opposite).

I can make the fixes for this.

K

**Comment by kparmar1** [comment]
Hi Thomas,

I was wondering If you can test out the fix as I don't  have another OS up and running and configured at the moment (I will do but not just at this moment)?
 
https://github.com/bitcoinj/bitcoinj/pull/1056

Thanks.

**Comment by mikehearn** [comment]
OS X has a case insensitive file system.

**Comment by traber** [comment]
The fixed code works for me but presumably I did not trigger the execution of all SQL statements during my tests. This is why I also manually checked the fixed source code. In DatabaseFullPrunedBlockStore I found one left over of a camel case column name in the SQL statement provided by the INSERT_OPENOUTPUTS_SQL constant ("toAddress"). This is no problem since column names are not case sensitive in MySQL on any platform.

**Comment by kparmar1** [comment]
Yeah I looked it up and was surprised. I assumed it would be closer to Linux. Anyway I'm moving over to Ubuntu now and surprisingly I actually like the experience (after I got it all working- a little bit of a pain involved).
